### PR TITLE
add openhsi to the rosdistro

### PR DESCRIPTION
**Pull request by hakuturu583**
Signed-off-by: Masaya Kataoka <ms.kataoka@gmail.com>

Please add the following dependency to the rosdep database.

## Package name:

python-openhsi

## Package Upstream Source:

https://github.com/openhsi/openhsi/

## Purpose of using this:

I want to resolve openhsi package in rosdep.
So, I want to add python-openhsi package to risdistro.
openhsi is a pure pip package.

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://pypi.org/project/openhsi/
- Ubuntu: https://packages.ubuntu.com/
   - https://pypi.org/project/openhsi/
- Fedora: https://src.fedoraproject.org/browse/projects/
  - https://pypi.org/project/openhsi/

# Please Add This Package to be indexed in the rosdistro.

all, openhsi is not a ros/ros2 package.

# The source is here: 

https://github.com/openhsi/openhsi/

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -3107,6 +3107,19 @@ python-opengl:
     xenial: [python-opengl]
     yakkety: [python-opengl]
     zesty: [python-opengl]
+python-openhsi:
```
Comment (ivanpauno) [inline rosdep/python.yaml:3110]: Please check the [contributing guidelines](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-rules)

```suggestion
python-openhsi-pip:
```

If you instead want the python3 version, you should add:

```suggestion
python3-openhsi-pip:
```

(in that case, the key needs to be re-alphabetized)

File: rosdep/python.yaml [hunk 2]
```diff
@@ -3107,6 +3107,19 @@ python-opengl:
...
+  debian:
+    pip:
+      packages: [openhsi]
+  fedora:
+    pip:
+      packages: [openhsi]
+  gentoo:
+    pip:
+      packages: [openhsi]
```
Comment (ivanpauno) [inline rosdep/python.yaml:3119]: According to the contributing guidelines:

> As a reminder pip rules should not be used on Gentoo. See above for the Gentoo specific guidelines.

**Review by ivanpauno — CHANGES_REQUESTED** [review]

File: rosdep/python.yaml (hunk 2 above)
Comment (hakuturu583) [reply rosdep/python.yaml]: fixed in https://github.com/ros/rosdistro/pull/32533/commits/e929a52b8ee2d29789545cee1a2a5706bac22a0d

File: rosdep/python.yaml (hunk 1 above)
Comment (hakuturu583) [reply rosdep/python.yaml]: fixed in https://github.com/ros/rosdistro/pull/32533/commits/555f81f0dfecf3b7df493b9162867b36f890f31d

**Review by ivanpauno — APPROVED** [review]

**Review by audrow — APPROVED** [review]

**Pull request merged**